ui.go

```diff
@@ -1171,6 +1171,15 @@ func (ui *ui) readNormalEvent(ev tcell.Event, nav *nav) expr {
 				ui.menuBuf = nil
 				return draw
 			}
+			if tev.Modifiers() == tcell.ModCtrl && !strings.HasPrefix(val, "<c-") {
```

Comment:
You could put this above the block with <esc>, to allow the user to do <s-esc> or something. I'm not sure if there's any perspective from which that's any less safe.

Reply:
I think you meant the escape key here, to allow for mappings like `<s-esc>` and `<a-esc>`? I have done this now, thanks for the advice.

Reply:
Yes, that's right. I'm not sure why my message ended up mangled.